WalletWasabi/WabiSabi/ProtocolContants.cs

```diff
@@ -0,0 +1,19 @@
+using NBitcoin;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Threading.Tasks;
+using WalletWasabi.Logging;
+using WalletWasabi.WabiSabi.Backend.Rounds;
+
+namespace WalletWasabi.WabiSabi
+{
+	public static class ProtocolConstants
+	{
+		public const int CredentialNumber = 2;
+		public const ulong MaxAmountPerAlice = 4_300_000_000_000ul;
+		public const uint MaxVsizePerAlice = 250;
```

Comment:
What's the value for this constant? I just divided the old 1000 by 4 but i am not sure this is correct. @nothingmuch do you have a better number?

Reply:
at the moment I think the most sensible value for general purposes is 255 (see slack for calculations for the main tradeoff, which is how many users, max fan-out ratio), for p2wpkh only this allows up to 1:6 ratio between number of inputs and outputs and max 391 inputs. other values might be more appropriate in certain settings, so perhaps this should probably be a round parameter with a default value, not a constant. probably not worth worrying about for now.

Reply:
done. thank you